boa3/model/builtin/interop/storage/findoptionstype.py

```diff
@@ -28,7 +28,7 @@ def build(cls, value: Any = None) -> IType:
     @classmethod
     def _is_type_of(cls, value: Any):
         from boa3.builtin.interop.storage import FindOptions
-        return isinstance(value, (FindOptions, FindOptionsType))
+        return isinstance(value, (FindOptions, FindOptionsType, type(int)))
```

Comment:
I think the problem is in the [`|` operation](https://github.com/CityOfZion/neo3-boa/blob/development/boa3/model/operation/binary/logical/logicor.py) and not in the type itself.